rows_test.go

```diff
@@ -292,3 +292,30 @@ func TestRowsNextStructError(t *testing.T) {
 		t.Errorf("Next should return an error")
 	}
 }
+
+func TestRowsNextStructWithNestedSpaces(t *testing.T) {
+	rowsJson := `{
+        "query":{"query_id":"16FF2A0300ECA753"},
+        "meta":[{"name":"struct_col","type":"struct(` + "`a b`" + ` int, s struct(` + "`c d`" + ` timestamp))"}],
```

Comment:
Is this a formatting issue or is there a need to do string addition here?

Reply:
You basically cannot escape a backtick symbol(\`) in a backticked string